src/AST-Core/RBMethodNode.class.st

```diff
@@ -708,7 +708,7 @@ RBMethodNode >> statements: aCollection [
 
 { #category : #accessing }
 RBMethodNode >> stop [
-	^ self newSource size
+	^(self sourceCode ifNotNil: [self sourceCode] ifNil: [ self formattedCode ]) size
```

Comment:
marcus may be can rewrite it as 

(self sourceCode ifNotNil: [ :s | s ] ifNil: [ self formattedCode ]) size

Reply:
I committed an update with that